app/controllers/load_calculations_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LoadCalculationsController < ApplicationController
+  before_filter :find_testing_ground
+
+  def heat
+    render json: {
```

Comment:
... although I'm not against changing the format if there is a compelling case for doing it differently.